packages/react/src/components/logo/Logo.tsx

```diff
@@ -23,11 +23,31 @@ export const logoRuDark =
 
 export type LogoSize = 'full' | 'small' | 'medium' | 'large';
 
-export type LogoProps = {
+interface LogoHidden {
+  /**
+   * Aria hidden property
+   */
+  ariaHidden: true;
+  /**
+   * Alt text to apply to the logo
+   */
+  alt?: never | '';
+}
+
+interface LogoShown {
+  /**
+   * Aria hidden property
+   */
+  ariaHidden?: never | false;
   /**
```

Comment:
Love that typescript shows an error when related props conflict! ❤️ 

Have to ask: what  'never' does here when 'ariaHidden' can be undefined too?

Reply:
It works with undefined too, just tested it to make sure.

Reply:
And when it's undefined, at least Chrome doesn't add the attribute at all. Did this help? 😄

Reply:
Shouting from the table next to you even though it's not a nice practice: would it be beneficial / easier to have LogoProps extend native img props, e.g. `React.ComponentPropsWithoutRef<'img'>`? Then you wouldn't need these extra types. This practice is used quite a lot in HDS components.

Reply:
I was just wondering in my first comment is `'never' `needed at all, if the prop does not have to passed at all `ariaHidden?`

Reply:
Just made ariaHidden to native aria-hidden -property, tomorrow morning I'll push the fiiinal version :D

Reply:
BTW we do have shitloads of ariaHidden in icons etc. 🤦  and ariaLabel* too